package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@datadog/pprof",
-  "version": "1.0.0",
+  "version": "2.0.0-pre",
```

Comment:
@rochdev Does this work or does it need to stay as `1.0.0`?

Reply:
The change is actually more accurate to be more in line with dd-trace. The main branch contains breaking changes so it's actually a beta for the next major.